renderer-macos/lib/stubs/BriskCocoa.c

```diff
@@ -2,73 +2,19 @@
 #import "BriskApplicationDelegate.h"
 #import "BriskWindowDelegate.h"
 #import <caml/threads.h>
-
-dispatch_semaphore_t caml_thread_sema;
+#import <caml/unixsupport.h>
```

Comment:
I'm not sure we need this here.

Reply:
I meant the `#import <caml/unixsupport.h>` piece.

Reply:
right